packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -97,7 +97,12 @@ - (void)pickImageWithPHPicker:(int)maxImagesAllowed API_AVAILABLE(ios(14)) {
 
   self.maxImagesAllowed = maxImagesAllowed;
 
-  [self checkPhotoAuthorizationForAccessLevel];
+  BOOL usePhaAsset = [[_arguments objectForKey:@"requestFullMetadata"] boolValue];
```

Comment:
The type is `PHAsset`, not `PHAAsset`, and also ObjC style is to capitalize abbreviations even in variable names, so this, and the similar variables below, should be `usePHAsset` rather than `usePhaAsset`.